Identifiers.py

```diff
@@ -22,28 +22,70 @@ class HomePageIdentifiers(object):
     WHATISCANONIZERHEADING = (By.XPATH, '//*[@id="exampleAccordion"]/ul[1]/li[1]/a/span')
     CANONIZER_LOGO = (By.XPATH, '//*[@id="__next"]/div/header/div[1]/a/span/img')
 
+
 class RegisterPageIdentifiers(object):
```

Comment:
Create a New Folder "Name" Identifiers and create new classes there. It'll sort out the code and make it easier to manage